libraries/AC_Fence/AC_Fence.h

```diff
@@ -147,19 +147,19 @@ class AC_Fence
     Action get_action() const { return _action; }
 
     /// get_safe_alt - returns maximum safe altitude (i.e. alt_max - margin)
-    float get_safe_alt_max() const { return _alt_max - _margin; }
+    float get_safe_alt_max_m() const { return _alt_max_m - _margin_u_m; }
 
-    /// get_safe_alt_min - returns the minimum safe altitude (i.e. alt_min + margin)
-    float get_safe_alt_min() const { return _alt_min + _margin; }
+    /// get_safe_alt_min_m - returns the minimum safe altitude (i.e. alt_min + margin)
+    float get_safe_alt_min_m() const { return _alt_min_m + _margin_u_m; }
 
-    /// get_radius - returns the fence radius in meters
-    float get_radius() const { return _circle_radius.get(); }
+    /// get_radius_m - returns the fence radius in meters
+    float get_radius_m() const { return _circle_radius_m.get(); }
 
-    /// get_margin - returns the fence margin in meters
-    float get_margin() const { return _margin.get(); }
+    /// get_margin_u_m - returns the fence margin in meters
```

Comment:
```suggestion
    /// get_margin_u_m - returns the vertical fence margin in meters
```
... and we really should update the parameter docs!